src/rules/tryceratops/rules/error_instead_of_exception.rs

```diff
@@ -0,0 +1,68 @@
+use ruff_macros::derive_message_formats;
+use rustpython_ast::{Excepthandler, ExcepthandlerKind, Expr, ExprKind};
+
+use crate::ast::helpers::collect_call_path;
+use crate::ast::types::Range;
+use crate::ast::visitor;
+use crate::ast::visitor::Visitor;
+use crate::checkers::ast::Checker;
+use crate::define_violation;
+use crate::registry::Diagnostic;
+use crate::violation::Violation;
+
+define_violation!(
+    pub struct ErrorInsteadOfException;
+);
+impl Violation for ErrorInsteadOfException {
+    #[derive_message_formats]
+    fn message(&self) -> String {
+        format!("Use `logging.exception` instead of `logging.error`")
+    }
+}
+
+#[derive(Default)]
+/// Collect `logging.error`-like calls from an AST. Matches `logging.error`,
+/// `logger.error`, `self.logger.error`, etc., but not arbitrary `foo.error`
+/// calls.
```

Comment:
I decided to make this a bit stricter. I'm just erring on the side of false negatives these days a little bit, especially for checks like this that lack strong static verifiability. Sorry for the last minute change!